### PR TITLE
Doc/heroku

### PR DESCRIPTION
**Pull request by raksonibs**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,11 +9,13 @@
   "author": "Max Stoiber",
   "license": "MIT",
   "scripts": {
+    "start:production": "npm run prod:build && npm run start:prod",
+    "start": "npm run start:tunnel",
     "prebuild": "npm run test && npm run build:clean",
     "build:clean": "rimraf ./build/*",
-    "build": "cross-env NODE_ENV=production webpack --config internals/webpack/webpack.prod.babel.js --color -p",
+    "build": "NODE_ENV=production webpack --config internals/webpack/webpack.prod.babel.js",
+    "prod:build": "npm run build:clean && npm run build",
```
Comment (mxstbr) [inline package.json]: Lets change this to `build:prod` in line with the other comments!

File: package.json [hunk 2]
```diff
@@ -9,11 +9,13 @@
   "author": "Max Stoiber",
   "license": "MIT",
   "scripts": {
+    "start:production": "npm run prod:build && npm run start:prod",
```
Comment (mxstbr) [inline package.json]: Lets move this below the `start` command!

**Comment by mxstbr** [comment]
Solid start @raksonibs, thanks! Instead of having the Procfile already in there, I'd rather the doc explains how to add one. Not everybody has a need to deploy to Heroku, so having that there by default seems like a lot of unnecessary file space.

Two other small nitpicks and then we can merge this! Thanks so much!

*/cc @grabbou who might have some input about this*

**Comment by mxstbr** [comment]
Awesome, the doc looks perfect now!

Could you rebase onto master, resolve the conflicts and then we'll land this, thanks @raksonibs :fire:

**Comment by raksonibs** [comment]
Hey @mxstbr, I don't have write access to the repo however, so I can't actually push my merge!

**Comment by mxstbr** [comment]
Did it myself, I realized that it's actually a bit more involved because we changed the commands recently. Thanks @raksonibs, this is awesome! :fire:

**Comment by raksonibs** [comment]
np:) will work on other issues soon. Got a list of features anywhere?

**Comment by mxstbr** [comment]
Right up there, the "Issues" tab. :wink:

**Pull request merged**